sources/functions.py

```diff
@@ -478,9 +478,20 @@ def post_index(index, server=SEFARIA_SERVER):
 
 def hasTags(comment):
     mod_comment = removeAllTags(comment)
-    return mod_comment != comment 
-
-
+    return mod_comment != comment
+     
+@weak_connection
+def post_category(info, server=SEFARIA_SERVER):
+    url = server+'/api/category/'
+    return http_request(url, body={'apikey': API_KEY} ,json_payload=info, method="POST")
+    
+def add_category(root, parent, en_title, he_title, server=SEFARIA_SERVER):
```

Comment:
The signature should be:
`def add_category(en_title, he_title, path)`
There's no guarantee that every category path will only have 3 items! Let the user add the whole path as a list, no need for any assumptions here.